packages/ui/components/Footer.tsx

```diff
@@ -1,5 +1,41 @@
+import { ReactNode } from 'react'
+
+import { ArrowUpRight, Code, Discord, Twitter } from '@dao-dao/icons'
+
 export interface FooterProps {}
 
+const FooterItem = ({ icon, name }: { icon: ReactNode; name: string }) => (
+  <div className="flex flex-row justify-between items-center p-2 hover:bg-background-interactive-hover rounded-md cursor-pointer">
+    <div className="flex flex-row gap-2 items-center text-text-tertiary">
+      <div>{icon}</div>
+      <div>{name}</div>
+    </div>
+
+    <ArrowUpRight className="text-text-tertiary" height={12} width={12} />
+  </div>
```

Comment:
```suggestion
  <div className="flex flex-row justify-between items-center p-2 text-text-tertiary hover:bg-background-interactive-hover rounded-md cursor-pointer">
    <div className="flex flex-row gap-2 items-center">
      <div>{icon}</div>
      <div>{name}</div>
    </div>

    <ArrowUpRight height={12} width={12} />
  </div>
```

You can move the text color class up to the parent and let the children inherit if you would like.